src/app/src/AboutSlide.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import { Button, Flex, Heading, Text } from 'rebass';
+import styled from 'styled-components';
+
+const StyledAboutSlide = styled(Flex)`
+    height: 80vh;
+    padding: 1rem;
+
+    video {
+        margin: auto;
+        height: fit-content;
+        max-width: 100%;
+    }
+`;
+
+const StyledPlayButton = styled(Button)`
```

Comment:
Oops! Sorry if I wasn't clear before.I think we're only naming the top-level style `Styled__` aka `StyledAboutSlide` because of the otherwise name conflict with the component itself `AboutSlide`. This can just be `PlayButton`